libkbfs/backpressure_disk_limiter.go

```diff
@@ -785,6 +810,18 @@ func defaultGetFreeBytesAndFiles(path string) (int64, int64, error) {
 	return int64(freeBytes), int64(freeFiles), nil
 }
 
+func (bdl *backpressureDiskLimiter) simpleByteTrackerFromType(typ diskLimitTrackerType) (
```

Comment:
`simpleResourceTrackerFromType`?  As I noted with `usedBytes`, there's some confusion here.  Can we make them consistent?  I think we should go all in with either "resources" or "bytes".